<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hedm_calibration_options_dialog</class>
 <widget class="QDialog" name="hedm_calibration_options_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HEDM Calibration Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="view_grains_table">
     <property name="text">
      <string>View Grains Table</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="grid_layout">
     <item row="4" column="1" colspan="2">
      <widget class="QGroupBox" name="clobbering_group">
       <property name="title">
        <string>Clobbering</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="clobber_strain">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sets and fixes the stretch matrix to identity.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Clobber strain?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="clobber_centroid">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sets and fixes the grain position to the origin: (0, 0, 0).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Clobber centroid?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="clobber_grain_Y">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sets and fixes the grain Y to zero.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Clobber grain Y?</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="num_hkls_selected">
       <property name="text">
        <string>Number of hkls selected:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QGroupBox" name="refitting_group">
       <property name="title">
        <string>Refitting</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="refit_pixel_scale_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Refit pixel scale</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="do_refit">
          <property name="text">
           <string>Do refit?</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="refit_ome_step_scale_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Refit ome step scale</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="ScientificDoubleSpinBox" name="refit_pixel_scale">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals">
           <number>8</number>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
          <property name="value">
           <double>2.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="ScientificDoubleSpinBox" name="refit_ome_step_scale">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals">
           <number>8</number>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="choose_hkls">
       <property name="text">
        <string>Choose HKLs</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vertical_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>choose_hkls</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>hedm_calibration_options_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>hedm_calibration_options_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>do_refit</sender>
   <signal>toggled(bool)</signal>
   <receiver>refit_pixel_scale_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>do_refit</sender>
   <signal>toggled(bool)</signal>
   <receiver>refit_pixel_scale</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>do_refit</sender>
   <signal>toggled(bool)</signal>
   <receiver>refit_ome_step_scale_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>do_refit</sender>
   <signal>toggled(bool)</signal>
   <receiver>refit_ome_step_scale</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
